src/Handler/TagHandler.php

```diff
@@ -106,6 +106,10 @@ public function addTags(array $tags)
      * Invalidate cache entries that contain any of the specified tags in their
      * tag header.
      *
+     * The cache manager is told to invalidate all content with the specified tags
+     * and invalidation is dispatched on request termination. To immediately
```

Comment:
Not just request termination, right? Also command termination, so more generally, kernel termination.